bridge/server/utils/configuration.ts

```diff
@@ -1,37 +1,224 @@
-import { EnabledComponents, LogDestination } from './logger';
+import { execSync } from 'child_process';
+import { dirname, join } from 'path';
+import { fileURLToPath } from 'url';
+
+import { EnabledComponents, LogDestination, logger as log } from './logger';
 
 /**
  * Option to configure the Bridge-Server
  */
 export interface BridgeOption {
   logging?: LogOptions;
+  auth?: AuthOptions;
+  oauth?: OAuthOptions;
+  api?: APIOptions;
+  feature?: FeatureOption;
+  version?: string;
+  mode?: string;
+  mongo?: MongoOptions;
+}
+
+export enum EnvType {
```

Comment:
Why is there no env-type PROD? What is the mode when it's running in production? Is it DEV then? I'm confused.

Reply:
I need to redirect the question to @Kirdock since he implemented the first version of the code.

Reply:
We also have `production`. The env var is set in the Dockerfile